### DISCUSSION TITLE
Harmonic Minor Scale

### DISCUSSION THREAD

**Discussion by kylenowcrylater** [SynthstromAudible/DelugeFirmware#935] [Ideas]
I’d love to see a harmonic minor scale added. I can do so manually but afaik lose the ability to transpose easily and change the root and stuff, plus one wrong button combo and I’ll go back to major key.

Really I’d love to have a few more options (pentatonic, blues would be cool too) but harmonic minor is a common one I use all the time and is most similar to the other modes, it’s odd to be excluded

**Comment by manysounds** [comment]
10,000,000%
A number of scales should be added. No more than 8 total, really.
Thing is, we can always "fake" another common mode by starting on the mode note, i.e. D Dorian is C Major. 
I'd much prefer having the Harmonic Minor, the "Gypsy" minor with the flat-5, and a few others. Altered scales are definitely a mainstay for many composers and writers. My Turkish friends would be psyched for a "Phyrgian Dominant" :)
